### PR TITLE
Suggesting missing step

### PR DESCRIPTION
**Pull request by jorlopama**
Added missing step on the section to add a new onmicrosoft.com domain  (Step 2) - it was confusing for a couple of customers since it doesn't say that after step 1 ,  customers should select their current onmicrosoft.com domain in order to have the  option suggested in the next step .

### CODE REVIEW

**Review by jorlopama — COMMENTED** [review]
Submit review

File: microsoft-365/admin/setup/add-domain.md [hunk 1]
```diff
@@ -156,7 +156,9 @@ To add an onmicrosoft.com domain:
 
 1. Go to the Microsoft admin center, **Settings** > <a href="https://go.microsoft.com/fwlink/p/?linkid=834818" target="_blank">**Domains**</a>.
 
-2. On the **Overview** tab, select **Add onmicrosoft.com domain**.
+2. On the **Domains** page, select your current ***.onmicrosoft.com(Default)** domain.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/add-domain.md]:
```suggestion
2. On the **Domains** page, select your current ***.onmicrosoft.com (Default)** domain.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @jorlopama, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@efrene Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@efrene Gentle reminder. Thanks

**Pull request merged**